_test/ensure-stubs-compile.sh

```diff
@@ -30,21 +30,39 @@ for dir in $changed_exercises; do
 	cp -r $dir/tests $dir/tests.orig
 	cp $dir/src/lib.rs $dir/lib.rs.orig
 
-	# This sed serves two purposes:
-	# First, in Travis CI, we may have already compiled using the example solution.
-	# Edit the src/lib.rs file so that we surely recompile using the stub.
-	# Second, ensures that the stub compiles without warnings.
-	sed -i -e '1i #![deny(warnings)]' "$dir/src/lib.rs"
+  if [ -z "$CLIPPY" ]; then
```

Comment:
Potentially confusing. -z here, but -n below. Use one or the other consistently throughout, otherwise confuses reader.